packages/react-components/lib/tasks/types/compose-clean.tsx

```diff
@@ -0,0 +1,155 @@
+import { Autocomplete, TextField } from '@mui/material';
+import React from 'react';
+import type { TaskBookingLabel } from 'api-client';
+import { TaskDefinition } from '../create-task';
+
+export const DefaultComposeCleanTaskDefinition: TaskDefinition = {
+  taskDefinitionId: 'compose-clean',
+  taskDisplayName: 'Clean',
+  requestCategory: 'compose',
+};
```

Comment:
1. I think these default definitions should be moved closer to resource manager.
2. The resource manager allows configuring the entire `TaskDefinitoin`, including `requestCategeory`, but the default request category is always used in `CreateTaskForm`. It should respect the configured category, even if it is invalid or not allow configuring it.

Reply:
Split out "configuration" from "definition", where the configuration stays nearer to the resource manager, https://github.com/open-rmf/rmf-web/pull/925/commits/7f7eb6203121b1f1631da5978761d2b86a2eb4ec

this allows us to modify only the modifiable fields (display name for now)